samples/peripheral/lpuart/sample.yaml

```diff
@@ -5,14 +5,14 @@ tests:
   samples.peripheral.lpuart:
     build_only: true
     build_on_all: true
-    platform_allow: nrf52dk_nrf52832 nrf52833dk_nrf52833 nrf52840dk_nrf52840 nrf9160dk_nrf9160
+    platform_allow: nrf52dk_nrf52832 nrf52833dk_nrf52833 nrf52840dk_nrf52840 nrf9160dk_nrf9160ns
```

Comment:
Please update also documentation, it still says that the target for nRF9160 DK is `nrf9160dk_nrf9160`: https://github.com/nrfconnect/sdk-nrf/blob/e3716141c15d37f743bee03a025a2b5505288c32/samples/peripheral/lpuart/README.rst#L25